evaluate.cpp

```
@@ -548,6 +583,127 @@ BOOL Set_Up_Set_Strings (const int set_type,
+      {
+      iApplied--;
+      CSetPublishChange<TObject> & change = changes [iApplied];
+      if (change.pOld)
+        liveMap.SetAt (change.strName, change.pOld);
+      else
+        liveMap.RemoveKey (change.strName);
+      }
+    throw;
+    }
+
+  for (vector<pair<CString, TObject *> >::iterator it =
```

Clang (14.0.6)

[warning] variable 'it' is not initialized

(cppcoreguidelines-init-variables)